modules/thermal_hydraulics/joss_paper.md

```diff
@@ -28,29 +28,64 @@ Module (THM) is an optional MOOSE module that provides capabilities designed for
 thermal hydraulic systems. Its core capability lies in assembling a network of
 coupled components, for instance, pipes, junctions, valves, etc.
 
-THM is used as the foundation for the applications RELAP-7 [@relap7theory] and
+THM is used as the foundation for several applications, including RELAP-7 [@relap7theory] and
 Sockeye [@hansel2021sockeye], both developed at Idaho National Laboratory, which
 respectively simulate two-phase flow components and heat pipes.
 
+This paper briefly describes some of THM's systems and functionality of
+THM and gives a basic demonstration of its capability.
+
 # Capabilities
 
 ## Components System
 
-The `Components` system allows users to add "component"s to their
-simulation, which in general may perform several tasks:
-
-- Build meshes (1D, 2D, or 3D)
-- Add MOOSE variables
-- Add MOOSE objects
+The `Components` system allows users to add "component"s, which are very flexible
+in their use, but in general represent "pieces" of a simulation, which may couple
+together. Common uses for components include adding meshes (1D, 2D, or 3D), variables,
+physics (or other equations), and output. Components provide a higher level syntax that
+hides lower level MOOSE objects such as `Kernel`s, `BoundaryCondition`s, etc. While
+`Action`s can also be used to create a higher level syntax, components provide much more
+capability, due to the ability to interact between components. Also, components make
```

Comment:
> ...much more capability..."

I would say targeted capability - may be. 

Also the sentence sounds like there would not be an interaction between MOOSE systems, which is not true, you can interact, but in a different way.